docs/configuring-playbook-bridge-mautrix-Generic.md

```diff
@@ -0,0 +1,96 @@
+# Setting up a Generic Mautrix Bridge (optional)
+
+The playbook can install and configure bridges with mautrix (Currently twitter, facebook, instagram, signal, hangouts, googlechat)
+
+
+You can see each bridge features at https://github.com/mautrix/SERVICENAME/blob/master/ROADMAP.md
+
+To enable a bridge add
+
+
+```yaml
+matrix_mautrix_SERVICENAME_enabled: true
+```
+
+to your `vars.yml`
+
+There are some additional things you may wish to configure about the bridge before you continue.
+
+You can add
+
+```yaml
+matrix_admin: "@YOUR_USERNAME:{{ matrix_domain }}"
+```
+to 'vars.yml' to configure a user as administrator for all bridges, or you prefer to do it bridge by bridge you can configure it with
+
+```yaml
+matrix_mautrix_SERVICENAME_configuration_extension_yaml: |
+  bridge:
+    permissions:
+      '@YOUR_USERNAME:{{ matrix_domain }}': admin
+```
+
+Encryption support is off by default. If you would like to enable encryption, add the following to your `vars.yml` file:
+```yaml
+matrix_mautrix_SERVICENAME_configuration_extension_yaml: |
+  bridge:
+    encryption:
+      allow: true
+      default: true
+```
+
+
+Using both would look like
+
+```yaml
+matrix_mautrix_SERVICENAME_configuration_extension_yaml: |
+  bridge:
+    permissions:
+      '@YOUR_USERNAME:{{ matrix_domain }}': admin
+    encryption:
+      allow: true
+      default: true
+```
+
```

Comment:
```suggestion

## Setting the bot's username
```

It's helpful to add sections to this file, instead of having a wall of text with various things.

Feel free to do the same (adding sections) in other places in this file.